source/developers-guide/event-guide/index.md

```diff
@@ -0,0 +1,638 @@
+---
+layout: default
+title: Shopware events
```

Comment:
Please use "Event System"

"Shopware events" sounds like more like event in terms like a happening.